src/Helpers/Click/ClickedEditArtwork.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  ActionType,
+  ClickedEditArtwork,
+  ContextModule,
+  PageOwnerType,
+} from "../../Schema"
+
+export interface ClickedEditArtworkArgs {
+  contextPageOwnerType: PageOwnerType
+  contextModule: ContextModule
+  destinationPageOwnerType: PageOwnerType
+  destinationPageOwnerId: string
+  destinationPageOwnerSlug: string
+  destinationPath: string
+}
+
+/**
+ * A user clicks to edit an artwork on web.
+ *
+ *  @example
+ *  ```
+ *  clickedExpansionToggle({
+ *    contextModule: ContextModule.toDoList,
+ *    contextPageOwnerType: OwnerType.home,
+ *    destinationPageOwnerType: OwnerType.artwork,
```

Comment:
@brendamanganelli question I had while working on this: do you think we need separate OwnerTypes for ArtworkEdit and/or CMSHome since those are distinct from artworks/home on artsy.net?

Reply:
I don't think we need to differentiate them, as we are keeping everything pretty generic naming-wise!

Reply:
Sounds good!